addons/redirect-mobile-forums/addon.json

```
@@ -1,6 +1,6 @@
 {
-  "name": "Redirect to standard forums",
-  "description": "Redirects you to the standard (\"main\") forums when you click a link to the mobile forums.",
+  "name": "Fix mobile forum links",
+  "description": "Redirects you to the standard (\"main\") forums when you click a broken link to the mobile forums.",
```

Not completely sure about changing the name/description to reflect the broken mobile view. This wording seems to imply that at least some links are broken when in fact, _all_ links to the mobile forums are broken. Plus, the point of this addon is to redirect you to the standard forums nonetheless, so it makes sense to focus on that functionality. I agree on everything else, though.

---

> mention that all links to the mobile forums don't work anymore?

Yeah, it would actually probably be a good idea to still note that. Here's my proposal:
```suggestion
  "name": "Redirect to standard forums",
  "description": "Redirects you to the standard (\"main\") view of the forums when you follow a link to the mobile forums (which don't work anymore).",
```

---

@DNin01 Sounds good but you mixed singular and plural in the last sentence

---

ugh good point